d2renderers/d2sketch/sketch.go

```diff
@@ -802,6 +802,13 @@ func ArrowheadJS(r *Runner, arrowhead d2target.Arrowhead, stroke string, strokeW
 			stroke,
 			BG_COLOR,
 		)
+	case d2target.CircleArrowhead:
+		arrowJS = fmt.Sprintf(
+			`node = rc.circle(-2, 0, 8, { strokeWidth: %d, stroke: "%s", fill: "%s", fillStyle: "solid", fillWeight: 1, seed: 5 })`,
```

Comment:
I don't think these coordinates are right. The test looks like the line is going to the top left of the circle instead of middle.

Reply:
![Screenshot 2024-05-18 at 14 01 06](https://github.com/terrastruct/d2/assets/63919918/e7bb5e07-a4ec-467d-8399-e20703e91aee)
Should be centered now.